cli/pcluster/config/update_policy.py

```diff
@@ -120,10 +132,41 @@ def _check_min_count(change, patch):
     return new_min >= old_min and new_max - new_min >= old_max - old_min
 
 
+def _is_bucket_pcluster_generated(stack_name):
+    params = utils.get_stack(stack_name).get("Parameters")
+    return utils.get_cfn_param(params, "RemoveBucketOnDeletion") == "True"
+
+
+def _check_generated_bucket(change, patch):
+
+    # If bucket is generated (no cluster_resource_bucket specified when creating) and no change in config
+    # Old value is retrieve from CFN's ResourcesS3Bucket param, and new value is from config(not specified)
+    # Diff based on config should be:
+    #
+    # old value                         new value
+    # ---------------------------------------------
```

Comment:
minor: I'd remove all these comments related to the visualization layer in this function.